pyproject.toml

```diff
@@ -13,6 +13,7 @@ ignore = [
   # temporary disabled until we fix them:
   "T201",
   "ANN",
+  "S101", # use of assert detected - using assert in tests is fine
```

Comment:
Instead please do this https://github.com/ansible/ansible-lint/blob/main/pyproject.toml#L274-L275

... it will save us from inserting asserts inside src/